parameters/parameterizing-extracts.md

```diff
@@ -0,0 +1,43 @@
+# Parameterizing Extracts 
```

Comment:
I know we've been inconsistent internally, but let's do load and save rather than extract and save.